### PR TITLE
PRETTY EVERYTHING

### PR DESCRIPTION
**Pull request by mixmix**
noticed prettify was only running on some files. Fixed the globs yo

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -77,8 +77,8 @@
     "test:only": "if grep -r --exclude-dir=node_modules --exclude-dir=.git --color 'test\\.only' ; then exit 1; fi",
     "test:bail": "npm run test:raw | tap-arc --bail",
     "test": "npm run test:raw | tap-arc && npm run test:only",
-    "format-code": "prettier --write \"*.js\"",
-    "format-code-staged": "pretty-quick --staged --pattern \"*.js\"",
+    "format-code": "prettier --write \"**/*.js\"",
```
Comment (mixmix) [inline package.json:80]: this jank

**Comment by mixmix** [comment]
I'm not sure what I hate more - prettier or inconsistent formatting :cry:

**Comment by mixmix** [comment]
https://www.npmjs.com/package/prettier-standard

**Comment by staltz** [comment]
LGTM - reviewed from the toilet 🚽

**Comment by mixmix** [comment]
WHO MERGED MY PR

# :cry: :hankey: :japanese_ogre: 

_it's good, I like not having to come back to mini-PRs_

**Pull request merged**